### DISCUSSION TITLE
I found that manager is initialized in PHP_MINIT_FUNCTION.

### DISCUSSION THREAD

**Discussion by jmjoy** [SkyAPM/SkyAPM-php-sdk#350] [Q&A] [answered]
I found that manager is initialized in PHP_MINIT_FUNCTION, manager is initialized with threads about skywalking login and heartbeat. But in linux, PHP_MINIT_FUNCTION will be called in every request, so skywalking login and heartbeat will be called in every request? This may consume performance.

**Comment by jmjoy** [comment, marked as answer]
@heyanlong Hi, I have doubts about this.

**Reply by heyanlong** [reply]
No, PHP_MINIT_FUNCTION only runs during the module loading phase

**Reply by jmjoy** [reply]
Yes, you are right.
